styleguide/screens/Patterns/Modals/Modals.js

```diff
@@ -0,0 +1,279 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C, Note, Placeholder } from '../../../components/Scaffold/Scaffold';
+import ModalAnimator from '../../../../components/Modal/ModalAnimator';
+import Window, { WindowTitle } from '../../../../components/Modal/Window';
+import WithCross from '../../../../components/Modal/WithCross';
+import Btn from '../../../../components/Btn/Btn';
+
+import shared from '../../../../styleguide/shared.css';
+import css from './Modals.css';
+import m from '../../../../globals/modifiers.css';
+
+export default class ModalDocumentation extends Component {
+  state = {
+    modalAnimator: false,
+    modalWindow: false,
+    modalWindowDark: false,
+    modalWithCross: false,
+    modalWithCrossDark: false,
+  };
+
+  toggleModalAnimator = () => {
+    this.setState(({ modalAnimator }) => ({
+      modalAnimator: !modalAnimator,
+    }));
+  };
+
+  toggleModalWindow = () => {
+    this.setState(({ modalWindow }) => ({
+      modalWindow: !modalWindow,
+    }));
+  };
+
+  toggleModalWindowDark = () => {
+    this.setState(({ modalWindowDark }) => ({
+      modalWindowDark: !modalWindowDark,
+    }));
+  };
+
+  toggleModalWithCross = () => {
+    this.setState(({ modalWithCross }) => ({
+      modalWithCross: !modalWithCross,
+    }));
+  };
+
+  toggleModalWithCrossDark = () => {
+    this.setState(({ modalWithCrossDark }) => ({
+      modalWithCrossDark: !modalWithCrossDark,
+    }));
+  };
+
+  render() {
+    return (
+      <div>
+        <H level={ 1 }>Modals</H>
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          Modals are a secondary window that allow you to display additional
+          information whilst keeping context. Please use sparingly, as modals
+          interrupt a users flow.
+        </T>
+        <D>
+          <H level={ 2 }>Modal Animator</H>
+          <T elm="p" className={ m.mtr }>
+            Use the modal animator to control how the modal window enters and leaves
+            the page. By default, the animator fades the window in.
+          </T>
+          <Note className={ m.mtr }>
+            <T elm="p">
+              We original wanted to have a more extravagant entrance animation for
```

Comment:
Maybe we could link to an article or stack-overflow post highlighting the issue here?

Reply:
I can't find anything else on this unfortunately... I went to make a demo of the issue and couldn't replicate it either 😂  I'll add the additional info as/when we find it 👍